docs/contributing/doc_contribution.md

```diff
@@ -34,6 +34,7 @@ Here are some VSCode Extensions can help you write and edit your markdown files
 
 - [Excel to Markdown Table](https://tableconvert.com/excel-to-markdown) - This website will convert an Excel table into markdown format, which can be copied and pasted into your markdown file.
 - [Material for MkDocs Reference](https://squidfunk.github.io/mkdocs-material/reference/) - This is the official reference for the Material for MkDocs theme, which will help you understand how to use the theme's features.
+- [Broken Link Check](https://www.brokenlinkcheck.com/) - This website will scan your website to ensure that all links are working correctly. This will only work on the deployed version of the documentation, not the local version.
```

Comment:
nice tool for partners, though I was not able to get it to function with our internal docs. "Subfolders / URLs with slashes are not supported in the FREE version." That said, I think this is valuable to highlight to folks the need to confirm their links are functional.

Reply:
I was able to get this to work with our docs, though?

Reply:
huh, weird. i guess it changed since I made this PR lol